### PR TITLE
Event View

### PR DESCRIPTION
**Pull request by morganecf**
Adds ability to display custom party events. Events are added to an `overrides/events.json` config.

Calendar button shows up next to map if there are events:
Screen Shot 2020-06-06 at 3 04 27 PM

Clicking on the button opens up an overlay:
Jun-06-2020 15-03-46

### CODE REVIEW

File: app/client/jsx/Room.jsx [hunk 1]
```diff
@@ -221,14 +230,22 @@ class Room extends Component {
...
                 <Beforeunload onBeforeunload={this.handleBeforeUnload.bind(this)} />
                 <Modal
                     isOpen={this.state.showMap}
                     onAfterOpen={this.handleOpenMap.bind(this)}
                     onRequestClose={this.handleCloseMap.bind(this)}>
                         <Map onRoomClick={this.onSwitchRoom.bind(this)} handleCloseMap={this.handleCloseMap.bind(this)}></Map>
                 </Modal>
+                <Modal
+                    isOpen={this.state.showEvents}
+                    onAfterOpen={this.handleOpenEvents.bind(this)}
```
Comment (SeanMcGrath) [inline app/client/jsx/Room.jsx:244]: am i reading this correctly that clicking the button triggers `handleOpenEvents` but we're also calling in `onAfterOpen`?

File: app/client/styles/components/_eventList.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+.ReactModal__Content.event-modal {
+    position: absolute;
+    width: 50%;
+    max-height: 85%;
+    left: 25%;
+    top: 40px;
+    border-radius: 5px;
+    outline: none;
+    overflow-y: auto;
+}
```
Comment (SeanMcGrath) [inline app/client/styles/components/_eventList.scss:59]: newline at EOF

File: app/config.py [hunk 3]
```diff
@@ -44,6 +49,13 @@ def ADVENTURES(self):
             return self.merged_cfg['adventures']
         except:
             return dict()
+
+    @property
+    def EVENTS(self):
+        try:
+            return self.merged_cfg['events']
```
Comment (SeanMcGrath) [inline app/config.py:56]: i think it's my bad for doing `try/except` here originally, can we not do `merged_cfg.get('events', dict())`?

Comment (morganecf) [reply app/config.py:56]: yes :)

File: app/client/jsx/Room.jsx (hunk 1 above)
Comment (morganecf) [reply app/client/jsx/Room.jsx:244]: Hm...yeah that doesn't make sense. I don't think we need that callback.

Comment (morganecf) [reply app/client/jsx/Room.jsx]: FYI I removed this callback but something funky happens when I do a similar thing for the map modal. Will look into this later.

**Pull request merged**